developers/README.md

```diff
@@ -17,6 +17,44 @@ Supported development platforms:
 * [Ubuntu](dev-environment/ubuntu.md)
 * [Apple's macOS](dev-environment/macOS.md)
 
+### Virtual Machines
```

Comment:
@mlangbehn hmm, I kind of like how this section links back to the Development Environment section on the same page. Maybe when we add Docker stuff to this page we can separate them out?